plugins/brain/__init__.py

```diff
@@ -2221,6 +2225,314 @@ def _inject_brain_secrets(ctx):
             fob.brain_config.similarity_backends[_backend][_key] = value
 
 
+class FindExactDuplicates(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="find_exact_duplicate_images",
+            label="Find exact duplicates",
+            light_icon="/assets/icon-light.svg",
+            dark_icon="/assets/icon-dark.svg",
+            description="Find exact duplicate images in the dataset",
+            dynamic=True,
+        )
+
+    def resolve_input(self, ctx: foo.ExecutionContext) -> types.Property:
+        inputs = types.Object()
+        form_view = types.View(
+            label="Find exact duplicates",
+            description="Find exact duplicates in the dataset",
+        )
+        return types.Property(inputs, view=form_view)
+
+    def execute(self, ctx: foo.ExecutionContext) -> dict[str, int]:
+
+        response = find_exact_duplicates(ctx.dataset)
+
+        if not ctx.delegated:
+            ctx.trigger("reload_dataset")
+        return response
+
+    def resolve_output(self, ctx: foo.ExecutionContext) -> types.Property:
```

Comment:
Note to self: we don't use `resolve_output()` in other core plugins. Feeling compelled to do this *might* be an indication that these operators should be in their own plugin? 🤔